demoshell/main.py

```diff
@@ -137,6 +148,31 @@ def extend_text(self, style, text):
     def received_output(self, data, style):
         self.extend_text(style, data.decode('utf-8'))
 
+    def _load_config(self):
+        appname = "DemoShell"
+        appauthor = "Doug Hellman"    
+        data_dir = user_data_dir(appname,appauthor)
+        filename = "demoshell.ini"
+        filepath = os.path.join(data_dir,filename)
+
+        if (os.path.isdir(data_dir) == False ):
+            os.makedirs(data_dir)
+        
+        if (os.path.isfile(filepath) == False):
+            with open(os.path.join(data_dir,filename),'w+') as configfile:
+                Config = configparser.ConfigParser()
+                Config.add_section('Aliases')
+                #Config.set("Aliases","ll","ls -la")  #This is how you would set alias with code. 
+                Config.write(configfile)    
+
+         
+    #Read Aliases below
+        with open(filepath,'r+') as configfile:
+            Config = configparser.ConfigParser()
+            Config.read(filepath)
+            for key in Config['Aliases'].keys():
+                self._aliases[key] = Config.get('Aliases',key)
```

Comment:
Instead of copying the alias definitions into another dictionary, could we just access the config object directly?